.github/workflows/build-cli-docker.yml

```diff
@@ -0,0 +1,166 @@
+---
+name: Build bws Docker image
+
+on:
+  push:
+    paths:
+      - "crates/bws/**"
+  workflow_call:
+    inputs:
+      sdk_branch:
+        type: string
+        default: master
+      is_workflow_call:
+        type: boolean
+        default: true
+  workflow_dispatch:
+    inputs:
+      sdk_branch:
+        description: "Server branch name to deploy (examples: 'master', 'rc', 'feature/sm')"
+        type: string
+        default: master
+  pull_request:
+    paths:
```

Comment:
Adding `Dockerfile` here would be beneficial to check if changes in dockerfile result in building an image in a PR.